### PR TITLE
[FLINK-36952][source-connector/mongodb] Fix MongoDB connector Chinese documentation

### PR DESCRIPTION
**Pull request by Thorne-coder**
Desc:[FLINK-36952](https://issues.apache.org/jira/browse/FLINK-36952)

### CODE REVIEW

**Comment by Thorne-coder** [comment]
@Jiabao-Sun @yuxiqian  PTAL

File: docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ SELECT * FROM products;
 **请注意**
 
 MongoDB 的更改事件记录在消息之前没有更新。因此，我们只能将其转换为 Flink 的 UPSERT 更改日志流。
-upstart 流需要一个唯一的密钥，所以我们必须声明 `_id` 作为主键。
+因为 upsert 流需要一个唯一的密钥，所以我们必须声明 `_id` 作为主键。
```
Comment (yuxiqian) [inline docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md:150]:
```suggestion
因为 upsert 流需要唯一键，所以我们必须声明 `_id` 作为主键。
```

**Review by yuxiqian — APPROVED** [review]
Thanks for @Thorne-coder's fix!

File: docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md [hunk 2]
```diff
@@ -367,7 +367,7 @@ MongoDB CDC 连接器是一个 Flink Source 连接器，它将首先读取数据
 
 ### 启动模式<a name="启动模式" id="002" ></a>
 
-配置选项```scan.startup.mode```指定 MySQL CDC 使用者的启动模式。有效枚举包括：
+配置选项```scan.startup.mode```指定 MongoDB CDC 使用者的启动模式。有效枚举包括：
```
Comment (Jiabao-Sun) [inline docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md:370]:
```suggestion
配置选项```scan.startup.mode```指定 MongoDB CDC 消费者的启动模式。有效枚举包括：
```

File: docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md (hunk 1 above)
Comment (Thorne-coder) [reply docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md]: thank u for u review and i fix it .

**Review by Jiabao-Sun — APPROVED** [review]
Thanks @Thorne-coder for this fix.
LGMT.

**Pull request merged**